contribution_plan/models.py

```diff
@@ -66,31 +44,12 @@ def filter(self, *args, **kwargs):
         return super(ContributionPlanManager, self).filter(*args, **kwargs)
 
 
-class ContributionPlan(core_models.UUIDVersionedModel):
-    id = models.AutoField(db_column='ContributionPlanId', primary_key=True)
-    uuid = models.CharField(db_column='ContributionPlanUUID', max_length=36, default=uuid.uuid4, unique=True)
-    version = models.IntegerField()
-
-    active = models.BooleanField(db_column='Active')
-
-    code = models.CharField(db_column='ContributionPlanCode', max_length=255, blank=True, null=True)
-    # calculation = models.ForeignKey(Calculation, db_column="CalculationUUID", on_delete=models.deletion.DO_NOTHING)
+class ContributionPlan(core_models.HistoryBusinessModel):
+    code = models.CharField(db_column="Code", max_length=255, blank=True, null=True)
+    name = models.CharField(db_column="Name", max_length=255, blank=True, null=True)
+    #calculation = models.ForeignKey(Calculation, db_column="CalculationUUID", on_delete=models.deletion.DO_NOTHING)
     benefit_plan = models.ForeignKey(Product, db_column="BenefitPlanUUID", on_delete=models.deletion.DO_NOTHING)
```

Comment:
need to update to id (as user) in next PR

Reply:
Ok, clear for me. Thx